### PR TITLE
Dashboard component

### PR DESCRIPTION
**Pull request by samkester**
- created dashboard component for the landing page after logging in
- dashboard requests user status from the backend using React Query
- added basic styles for readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,137 @@
...
-- point two
-- point three
-
-🚫 List the rest of the back end end features and libraries in the same format as the framework above
+- Users and organizations can create and update public-facing profiles.
+- Users can apply to join organizations as a microentrepreneur.
+- Users can apply to create a new partner organization.
+- Admins for an organization can accept or reject applications.
 
 # APIs
```
Comment (arn-foto) [inline README.md:23]: This endpoints are very clear and easy to understand, awesome!

File: src/components/pages/Home/RenderHomePage.js [hunk 2]
```diff
@@ -1,34 +1,46 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (arn-foto) [inline src/components/pages/Home/RenderHomePage.js:1]: Very clean looking formatting. Its easy to understand what this code is doing.

**Review by arn-foto — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JeanFraga) [reply README.md:23]: Great job!

File: src/components/pages/Home/RenderHomePage.js (hunk 2 above)
Comment (JeanFraga) [reply src/components/pages/Home/RenderHomePage.js:1]: looks good!

**Review by JeanFraga — APPROVED** [review]

**Pull request merged**